### PR TITLE
Additional trust bundle

### PR DESCRIPTION
**Pull request by Avielyo10**

### CODE REVIEW

**Comment by Avielyo10** [comment]
@zaneb According to your CR, this also fix a bug caused when passing the ca through env vars

**Comment by Avielyo10** [comment]
@hardys @zaneb

**Comment by hardys** [comment]
Looks like you need to rebase this on latest master since https://github.com/openshift-metal3/dev-scripts/pull/1152 merged?

**Comment by Avielyo10** [comment]
@hardys I merged

**Comment by hardys** [comment]
> @hardys I merged

Please rebase so we can review only the new changes e.g

```
git checkout master
git pull
git checkout additional-trust-bundle
git rebase -i origin
```

File: ocp_install_env.sh [hunk 1]
```diff
@@ -98,16 +98,14 @@ EOF
...
-EOF
-    elif [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ ! -z "$MIRROR_IMAGES" ]; then
-cat <<EOF
-$(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
-EOF
+  if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ]; then
+    if [ -z $MIRROR_IMAGES ]; then
+      echo "additionalTrustBundle: |"
     fi
+cat << EOF
```
Comment (zaneb) [inline ocp_install_env.sh:105]: This cat is unnecessary, it makes this really hard to read for no reason.

Actually the other one isn't necessary either. All you need is:

```
awk '{ print " ", $0 }' "${ADDITIONAL_TRUST_BUNDLE}"

Comment (Avielyo10) [reply ocp_install_env.sh]: Done

**Comment by Avielyo10** [comment]
@hardys @zaneb done.

**Comment by Avielyo10** [comment]
@hardys @zaneb Can someone take a look?

**Comment by Avielyo10** [comment]
@zaneb @hardys @vrutkovs

**Comment by hardys** [comment]
> @zaneb @hardys @vrutkovs

Please can you rebase and squash the commits like I previously asked?

**Comment by Avielyo10** [comment]
@hardys done

**Comment by hardys** [comment]
/ok-to-test
/approve

Thanks!

**Pull request merged**